test/apps/iodemo/io_demo.cc

```diff
@@ -785,13 +786,71 @@ class P2pDemoCommon : public UcxContext {
         return (data_size + chunk_size - 1) / chunk_size;
     }
 
+    static void handle_validate_error(const std::exception &e,
+                                      const UcxConnection *conn) {
+        LOG << "\"" << e.what() << "\" detected on " << conn->get_log_prefix()
+            << " (status=" << ucs_status_string(conn->ucx_status()) << ")";
+        abort();
+    }
+
+    class ValidateError : public std::exception {
+    public:
+        ValidateError() throw() {
+        }
+
+        ValidateError(const ValidateError& other) throw() {
+        }
+
+        ~ValidateError() throw() {
+        }
```

Comment:
why needed?

Reply:
compiling it w/o c++11 support requires it

Reply:
empty ctors/dtor?

Reply:
yes
```
io_demo.cc:796:11: error: looser throw specifier for ‘virtual P2pDemoCommon::ValidateError::~ValidateError()’
     class ValidateError : public std::exception {
           ^
In file included from /usr/include/c++/4.8.2/new:40:0,
                 from /usr/include/c++/4.8.2/ext/new_allocator.h:33,
                 from /usr/include/c++/4.8.2/x86_64-redhat-linux/bits/c++allocator.h:33,
                 from /usr/include/c++/4.8.2/bits/allocator.h:46,
                 from /usr/include/c++/4.8.2/deque:61,
                 from ucx_wrapper.h:15,
                 from io_demo.cc:7:
/usr/include/c++/4.8.2/exception:64:13: error:   overriding ‘virtual std::exception::~exception() throw ()’
     virtual ~exception() _GLIBCXX_USE_NOEXCEPT;
```